File: prelab11/mystery6.py
def mystery6(n,k):
	if (k ==0 or k==n):
		return 1
	elif():
		return 0
	else:
		return mystery6(n-1,k-1) + mystery6(n-1,k)

print(mystery6(4,3))
print(mystery6(4,2))
print(mystery6(4,3))
print(mystery6(5,3))
print(mystery6(5,4))

File: prelab11/mystery6.java
public static int mystery6(int n, int k) {
    if (k == 0 || k == n) {
        return 1;
    } else if (k > n) {
        return 0;
    } else {
        return mystery6(n - 1, k - 1) + mystery6(n - 1, k);
    }
}

System.out.print(mystery6(7,1));
